github-repo/action.yaml

```diff
@@ -29,31 +25,24 @@ inputs:
       Sets the score threshold for scans. Scores that fall below the threshold will exit 1. (default "0" - job continues regardless of the score returned by a scan).
     default: "0"
     required: false
-  service-account-credentials:
-    description: "Base64 encoded service account credentials used to authenticate with Mondoo Platform"
-    required: true
   is-cicd:
     description: "Automatically detects the GitHub Action Runs and report results into the CI/CD view"
     default: true
 runs:
-  using: "composite"
-  steps:
-    - name: Install Mondoo Client
-      shell: bash
-      run: |
-        echo Installing Mondoo Client...
-        echo ${{ inputs.service-account-credentials }} | base64 -d > mondoo.json
-        curl -sSL https://mondoo.com/install.sh | bash
-    - name: Mondoo status
-      shell: bash
-      run: mondoo status --config mondoo.json
-    - name: Run mondoo scan ${{ inputs.asset-type }}
-      shell: bash
-      run: >
-        mondoo scan github repo ${{ inputs.repository }}  --token "${{ inputs.token }}"
-        --log-level "${{ inputs.log-level }}"
-        --output "${{ inputs.output }}"
-        --score-threshold "${{ inputs.score-threshold }}"
-        --detect-cicd="${{ inputs.is-cicd }}"
-        --config mondoo.json
-        ${{ inputs.args }}
+  using: "docker"
+  image: "docker://mondoo/cnspec:7"
+  args:
+    - scan
+    - github
+    - repo
+    - ${{ inputs.repository }}
+    - --token
+    - ${{ inputs.token }}
```

Comment:
can we update this to use the `env` blog and set the token with the `GITHUB_TOKEN` environment variable? We should encourage users to use the ENV variable supported by `cnspec`

Reply:
Done